tests/golden/check-ignored/check-ignored.bats

```diff
@@ -39,3 +39,31 @@ load '../helpers'
 
   assert_diff expected.gold
 }
+
+@test "Config: Absolute fiepath in \"ignored\" error" {
+  run xrefcheck --config ./config-ignored-bad-path-absolute.yaml
+
+  assert_failure
+  assert_output --partial "Expected a relative glob, but got /to-ignore/inner-directory/broken_annotation.md"
+}
+
+@test "Config: Malformed glob in \"ignored\" error" {
+  run xrefcheck --config ./config-ignored-malformed-glob.yaml
+
+  assert_failure
+  assert_output --partial "compile :: bad <>"
+}
+
+@test "CLI: Absolute filepath in \"ignored\" yields to error" {
+  run xrefcheck\
+    --ignored "/to-ignore/*"
+  assert_failure
+  assert_output --partial "option --ignored: Expected a relative glob, but got /to-ignore/*"
+}
+
+@test "CLI: Malformed glob in arg \"ignored\" yields to error" {
+  run xrefcheck\
+    --ignored "<to-ignore>"
+  assert_failure
+  assert_output --partial "option --ignored: compile :: bad <>"
```

Comment:
I guess we could enrich the error message here a bit. 

When we fail to compile a glob pattern, we could tell the user to consult the syntax for glob patterns here: https://hackage.haskell.org/package/Glob/docs/System-FilePath-Glob.html#v:compile

---

We could also tell the user how to escape characters (as far as I can tell, this isn't documented anywhere). I found [a trick here](https://stackoverflow.com/a/2595162/857807).

Basically, when you have a file literally named `<a>.md` and you want to ignore it, `xrefcheck --ignored "<a>.md"` will fail to compile the pattern.

But if you wrap special characters in `[` and `]`, it'll work as expected: `xrefcheck --ignored "[<]a[>].md"
`

Reply:
Done